includes/pattern.inc

```diff
@@ -162,6 +162,7 @@ function oe_bootstrap_theme_preprocess_pattern_featured_media(&$variables) {
  * Implements hook_preprocess_HOOK() for inpage navigation pattern.
  */
 function oe_bootstrap_theme_preprocess_pattern_inpage_navigation(&$variables): void {
+  $variables['inpage_navigation_dropdown_id'] = $variables['attributes']['id'] . '-dropdown' ?? Html::getUniqueId('bcl-inpage-navigation') . '-dropdown';
```

Comment:
We always need to run everything through getUniqueId().
```suggestion
  $variables['inpage_navigation_dropdown_id'] = Html::getUniqueId(($variables['attributes']['id'] ?? 'bcl-inpage-navigation') . '-dropdown');
```

Reply:
I checked locally and indeed this pops up 
![image](https://github.com/user-attachments/assets/f276df75-3714-43ba-989a-b38974fb0093)

Reply:
done, all the points.